package.json 수정시 아래와 같은 에러가 발생합니다.
```
4:07:26 PM: sh: 1: husky: not found
4:07:26 PM: npm ERR! code 127
4:07:26 PM: npm ERR! path /opt/build/repo
4:07:26 PM: npm ERR! command failed
4:07:26 PM: npm ERR! command sh -c husky i
```
husky prepare 명령어 삭제 후 재시도 예정입니다.

--- a/.eslintrc.json
+++ b/.eslintrc.json
@@ -6,11 +6,7 @@
     "import/no-extraneous-dependencies": [
       "error",
       {
-        "devDependencies": [
-          "**/*.spec.jsx",
-          "**/*.spec.js",
-          "./src/app/store.js"
-        ]
+        "devDependencies": ["**/*.spec.jsx", "**/*.spec.js"]
       }
     ],
     "no-console": [

--- a/package.json
+++ b/package.json
@@ -30,6 +30,7 @@
     "react-redux": "^7.2.5",
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
+    "redux-logger": "^3.0.6",
     "styled-components": "^5.3.1",
     "styled-reset": "^4.3.4"
   },
@@ -42,7 +43,6 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "husky": "^7.0.0",
     "lint-staged": "^11.1.2",
-    "redux-logger": "^3.0.6",
     "stylelint": "^13.13.1",
     "stylelint-config-recommended": "^5.0.0",
     "stylelint-config-styled-components": "^0.1.1",

--- a/package.json
+++ b/package.json
@@ -19,6 +19,7 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.21.4",
+    "eslint-config-airbnb": "^18.2.1",
     "lodash.debounce": "^4.0.8",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
@@ -36,7 +37,6 @@
   },
   "devDependencies": {
     "eslint": "^7.32.0",
-    "eslint-config-airbnb": "^18.2.1",
     "eslint-plugin-import": "^2.24.2",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.26.1",

--- a/package.json
+++ b/package.json
@@ -7,7 +7,6 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "prepare": "husky install",
     "pre-commit": "lint-staged"
   },
   "repository": {

